modules/power-monitoring-accessing-dashboards-developer.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-accessing-dashboards-developer_{context}"]
+= Accessing {PM-shortname} dashboards as a developer
+
+You can access {PM-shortname} dashboards from the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the {product-title} web console.
+* You have access to the cluster as a developer or as a user.
+* You have installed the {PM-operator}.
+* You have deployed {PM-kepler} in your cluster.
+* You have enabled monitoring for user-defined projects.
+* You have `view` permissions for the namespace `openshift-power-monitoring`, the namespace where {PM-kepler} is deployed to.
+
+.Procedure
+
+. In the *Developer* perspective of the web console, go to *Observe* -> *Dashboard*.
+
+. From the *Dashboard* drop-down list, select the {PM-shortname} dashboard you want to see:
+** *Power Monitoring / Overview*
```

Comment:
I'm unsure if this is the only option or the expected option? Maybe it's just an example?

Reply:
**Overview** dashboard is the only one you can view currently from **Developer** perspective. It comes in a dropdown as there you have 2 dashboard to view in the **Administrator** perspective.

Reply:
Here's a screenshot for reference: 
![image](https://github.com/openshift/openshift-docs/assets/109141899/247d6ce3-73b1-4ec1-b4b5-edf038c3521e)

Reply:
OK, thanks for sharing. Is it me as I do not see " *Power Monitoring / Overview*"

Reply:
@dfitzmau Just confirmed with our QE @vprashar2929. The correct name for the dashboard should be **Power Monitoring/Overview**.
Here's the correct screenshot: 
![image](https://github.com/openshift/openshift-docs/assets/109141899/89727cab-e092-4312-9ac0-4191c0cc2597)

Reply:
Thanks. I now see this in the screenshot.